src/main/java/htsjdk/samtools/cram/CRAIEntry.java

```diff
@@ -7,27 +7,36 @@
 
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.List;
+import java.util.*;
+import java.util.stream.Collectors;
 
 /**
  * A class representing CRAI index entry: file and alignment offsets for each slice.
  * Created by vadim on 10/08/2015.
  */
-public class CRAIEntry implements Comparable<CRAIEntry>, Cloneable {
-    public int sequenceId;
-    public int alignmentStart;
-    public int alignmentSpan;
-    public long containerStartOffset;
-    public int sliceOffset;
-    public int sliceSize;
-    public int sliceIndex;
```

Comment:
Removed sliceIndex because it can't be represented in CRAI.  We were incorrectly using this uninitialized field to populate Slices.

Reply:
There are still references to slice.index in indexing code. Maybe for the BAI ?

Reply:
Yes, it's still used for BAI